from PyQt6 import QtCore, QtGui, QtWidgets
import resource_rc

class Ui_Level_Menu(object):
    def setupUi(self, Padavan_l1_2):
        Padavan_l1_2.setObjectName("Padavan_l1_2")
        Padavan_l1_2.resize(915, 678)
        Padavan_l1_2.setMinimumSize(QtCore.QSize(915, 678))
        Padavan_l1_2.setMaximumSize(QtCore.QSize(915, 678))
        icon = QtGui.QIcon()
        icon.addPixmap(QtGui.QPixmap(":/recurce/icon.png"), QtGui.QIcon.Mode.Normal, QtGui.QIcon.State.Off)
        Padavan_l1_2.setWindowIcon(icon)
        self.centralwidget = QtWidgets.QWidget(Padavan_l1_2)
        self.centralwidget.setMinimumSize(QtCore.QSize(915, 657))
        self.centralwidget.setStyleSheet("background-color: rgb(255, 255, 255);")
        self.centralwidget.setObjectName("centralwidget")
        self.gridLayout = QtWidgets.QGridLayout(self.centralwidget)
        self.gridLayout.setContentsMargins(15, 11, -1, 11)
        self.gridLayout.setObjectName("gridLayout")
        self.frame_3 = QtWidgets.QFrame(self.centralwidget)
        self.frame_3.setMinimumSize(QtCore.QSize(0, 50))
        self.frame_3.setMaximumSize(QtCore.QSize(16777215, 60))
        font = QtGui.QFont()
        font.setFamily("PT Sans")
        font.setPointSize(23)
        font.setBold(True)
        font.setWeight(75)
        self.frame_3.setFont(font)
        self.frame_3.setFrameShape(QtWidgets.QFrame.Shape.StyledPanel)
        self.frame_3.setFrameShadow(QtWidgets.QFrame.Shadow.Raised)
        self.frame_3.setObjectName("frame_3")
        self.verticalLayout_2 = QtWidgets.QVBoxLayout(self.frame_3)
        self.verticalLayout_2.setContentsMargins(0, 0, 0, 15)
        self.verticalLayout_2.setObjectName("verticalLayout_2")
        self.label_3 = QtWidgets.QLabel(self.frame_3)
        self.label_3.setMinimumSize(QtCore.QSize(0, 60))
        font = QtGui.QFont()
        font.setFamily("PT Sans")
        font.setPointSize(23)
        font.setBold(True)
        font.setWeight(75)
        self.label_3.setFont(font)
        self.label_3.setObjectName("label_3")
        self.verticalLayout_2.addWidget(self.label_3)
        self.gridLayout.addWidget(self.frame_3, 0, 0, 1, 2)
        self.frame_2 = QtWidgets.QFrame(self.centralwidget)
        self.frame_2.setMaximumSize(QtCore.QSize(16777215, 50))
        self.frame_2.setFrameShape(QtWidgets.QFrame.Shape.StyledPanel)
        self.frame_2.setFrameShadow(QtWidgets.QFrame.Shadow.Raised)
        self.frame_2.setObjectName("frame_2")
        self.horizontalLayout = QtWidgets.QHBoxLayout(self.frame_2)
        self.horizontalLayout.setContentsMargins(-1, 0, -1, 0)
        self.horizontalLayout.setObjectName("horizontalLayout")
        self.pushButton = QtWidgets.QPushButton(self.frame_2)
        self.pushButton.setMinimumSize(QtCore.QSize(0, 40))
        self.pushButton.setMaximumSize(QtCore.QSize(16777215, 40))
        self.pushButton.setStyleSheet("image: url(:/recurce/book.png);\n"
"border: none;")
        self.pushButton.setText("")
        self.pushButton.setObjectName("pushButton")
        self.horizontalLayout.addWidget(self.pushButton)
        self.pushButton_3 = QtWidgets.QPushButton(self.frame_2)
        self.pushButton_3.setMinimumSize(QtCore.QSize(0, 40))
        self.pushButton_3.setMaximumSize(QtCore.QSize(16777215, 40))
        self.pushButton_3.setStyleSheet("image: url(:/recurce/home.png);\n"
"border: none;")
        self.pushButton_3.setText("")
        self.pushButton_3.setObjectName("pushButton_3")
        self.horizontalLayout.addWidget(self.pushButton_3)
        self.gridLayout.addWidget(self.frame_2, 8, 0, 1, 2)
        self.frame = QtWidgets.QFrame(self.centralwidget)
        self.frame.setMinimumSize(QtCore.QSize(0, 500))
        self.frame.setFrameShape(QtWidgets.QFrame.Shape.StyledPanel)
        self.frame.setFrameShadow(QtWidgets.QFrame.Shadow.Raised)
        self.frame.setObjectName("frame")
        self.verticalLayout = QtWidgets.QVBoxLayout(self.frame)
        self.verticalLayout.setObjectName("verticalLayout")
        self.label = QtWidgets.QLabel(self.frame)
        font = QtGui.QFont()
        font.setFamily("PT Sans")
        font.setPointSize(22)
        font.setBold(True)
        font.setWeight(75)
        self.label.setFont(font)
        self.label.setObjectName("label")
        self.verticalLayout.addWidget(self.label)
        self.gridLayout_2 = QtWidgets.QGridLayout()
        self.gridLayout_2.setObjectName("gridLayout_2")
        self.pushButton_4 = QtWidgets.QPushButton(self.frame)
        self.pushButton_4.setMinimumSize(QtCore.QSize(40, 40))
        font = QtGui.QFont()
        font.setFamily("PT Sans")
        font.setPointSize(18)
        self.pushButton_4.setFont(font)
        self.pushButton_4.setStyleSheet("color: rgb(255, 255, 255);\n"
"background-color: rgb(95, 136, 240);\n"
"border-radius: 14px;")
        self.pushButton_4.setObjectName("pushButton_4")
        self.gridLayout_2.addWidget(self.pushButton_4, 1, 1, 1, 1)
        self.pushButton_6 = QtWidgets.QPushButton(self.frame)
        self.pushButton_6.setMinimumSize(QtCore.QSize(0, 40))
        font = QtGui.QFont()
        font.setFamily("PT Sans")
        font.setPointSize(18)
        self.pushButton_6.setFont(font)
        self.pushButton_6.setStyleSheet("color: rgb(255, 255, 255);\n"
"background-color: rgb(95, 136, 240);\n"
"border-radius: 14px;")
        self.pushButton_6.setObjectName("pushButton_6")
        self.gridLayout_2.addWidget(self.pushButton_6, 2, 0, 1, 1)
        self.pushButton_5 = QtWidgets.QPushButton(self.frame)
        self.pushButton_5.setMinimumSize(QtCore.QSize(0, 40))
        font = QtGui.QFont()
        font.setFamily("PT Sans")
        font.setPointSize(18)
        self.pushButton_5.setFont(font)
        self.pushButton_5.setStyleSheet("color: rgb(255, 255, 255);\n"
"background-color: rgb(95, 136, 240);\n"
"border-radius: 14px;")
        self.pushButton_5.setObjectName("pushButton_5")
        self.gridLayout_2.addWidget(self.pushButton_5, 3, 1, 1, 1)
        self.pushButton_2 = QtWidgets.QPushButton(self.frame)
        self.pushButton_2.setMinimumSize(QtCore.QSize(0, 40))
        self.pushButton_2.setMaximumSize(QtCore.QSize(16777215, 40))
        font = QtGui.QFont()
        font.setFamily("PT Sans")
        font.setPointSize(18)
        self.pushButton_2.setFont(font)
        self.pushButton_2.setLayoutDirection(QtCore.Qt.LayoutDirection.LeftToRight)
        self.pushButton_2.setStyleSheet("color: rgb(255, 255, 255);\n"
"background-color: rgb(95, 136, 240);\n"
"border-radius: 14px;")
        self.pushButton_2.setObjectName("pushButton_2")
        self.gridLayout_2.addWidget(self.pushButton_2, 1, 0, 1, 1)
        self.pushButton_16 = QtWidgets.QPushButton(self.frame)
        self.pushButton_16.setMinimumSize(QtCore.QSize(0, 40))
        font = QtGui.QFont()
        font.setFamily("PT Sans")
        font.setPointSize(18)
        self.pushButton_16.setFont(font)
        self.pushButton_16.setStyleSheet("color: rgb(255, 255, 255);\n"
"background-color: rgb(95, 136, 240);\n"
"border-radius: 14px;")
        self.pushButton_16.setObjectName("pushButton_16")
        self.gridLayout_2.addWidget(self.pushButton_16, 4, 0, 1, 1)
        self.pushButton_15 = QtWidgets.QPushButton(self.frame)
        self.pushButton_15.setMinimumSize(QtCore.QSize(0, 40))
        font = QtGui.QFont()
        font.setFamily("PT Sans")
        font.setPointSize(18)
        self.pushButton_15.setFont(font)
        self.pushButton_15.setStyleSheet("color: rgb(255, 255, 255);\n"
"background-color: rgb(95, 136, 240);\n"
"border-radius: 14px;")
        self.pushButton_15.setObjectName("pushButton_15")
        self.gridLayout_2.addWidget(self.pushButton_15, 3, 0, 1, 1)
        self.pushButton_7 = QtWidgets.QPushButton(self.frame)
        self.pushButton_7.setMinimumSize(QtCore.QSize(0, 40))
        font = QtGui.QFont()
        font.setFamily("PT Sans")
        font.setPointSize(18)
        self.pushButton_7.setFont(font)
        self.pushButton_7.setStyleSheet("color: rgb(255, 255, 255);\n"
"background-color: rgb(95, 136, 240);\n"
"border-radius: 14px;")
        self.pushButton_7.setObjectName("pushButton_7")
        self.gridLayout_2.addWidget(self.pushButton_7, 2, 1, 1, 1)
        self.pushButton_8 = QtWidgets.QPushButton(self.frame)
        self.pushButton_8.setMinimumSize(QtCore.QSize(0, 40))
        font = QtGui.QFont()
        font.setFamily("PT Sans")
        font.setPointSize(18)
        self.pushButton_8.setFont(font)
        self.pushButton_8.setStyleSheet("color: rgb(255, 255, 255);\n"
"background-color: rgb(95, 136, 240);\n"
"border-radius: 14px;")
        self.pushButton_8.setObjectName("pushButton_8")
        self.gridLayout_2.addWidget(self.pushButton_8, 4, 1, 1, 1)
        self.verticalLayout.addLayout(self.gridLayout_2)
        self.label_2 = QtWidgets.QLabel(self.frame)
        font = QtGui.QFont()
        font.setFamily("PT Sans")
        font.setPointSize(22)
        font.setBold(True)
        font.setWeight(75)
        self.label_2.setFont(font)
        self.label_2.setObjectName("label_2")
        self.verticalLayout.addWidget(self.label_2)
        self.label_6 = QtWidgets.QLabel(self.frame)
        font = QtGui.QFont()
        font.setFamily("PT Sans")
        font.setPointSize(18)
        self.label_6.setFont(font)
        self.label_6.setObjectName("label_6")
        self.verticalLayout.addWidget(self.label_6)
        self.label_5 = QtWidgets.QLabel(self.frame)
        font = QtGui.QFont()
        font.setFamily("PT Sans")
        font.setPointSize(22)
        font.setBold(True)
        font.setWeight(75)
        self.label_5.setFont(font)
        self.label_5.setObjectName("label_5")
        self.verticalLayout.addWidget(self.label_5)
        self.label_4 = QtWidgets.QLabel(self.frame)
        font = QtGui.QFont()
        font.setFamily("PT Sans")
        font.setPointSize(18)
        self.label_4.setFont(font)
        self.label_4.setObjectName("label_4")
        self.verticalLayout.addWidget(self.label_4)
        self.gridLayout.addWidget(self.frame, 1, 0, 1, 2)
        Padavan_l1_2.setCentralWidget(self.centralwidget)

        self.retranslateUi(Padavan_l1_2)
        QtCore.QMetaObject.connectSlotsByName(Padavan_l1_2)

        self.label.setStyleSheet("font-weight: bold")
        self.label_2.setStyleSheet("font-weight: bold")
        self.label_5.setStyleSheet("font-weight: bold")
        self.label_3.setStyleSheet("font-weight: bold")

    def retranslateUi(self, Padavan_l1_2):
        _translate = QtCore.QCoreApplication.translate
        Padavan_l1_2.setWindowTitle(_translate("Padavan_l1_2", "Level Menu"))
        self.label_3.setText(_translate("Padavan_l1_2", "<html><head/><body><p align=\"center\">Выбор уровня</p></body></html>"))
        self.label.setText(_translate("Padavan_l1_2", "Падаван"))
        self.pushButton_4.setText(_translate("Padavan_l1_2", "(в разработке)"))
        self.pushButton_6.setText(_translate("Padavan_l1_2", "(в разработке)"))
        self.pushButton_5.setText(_translate("Padavan_l1_2", "(в разработке)"))
        self.pushButton_2.setText(_translate("Padavan_l1_2", "1"))
        self.pushButton_16.setText(_translate("Padavan_l1_2", "(в разработке)"))
        self.pushButton_15.setText(_translate("Padavan_l1_2", "(в разработке)"))
        self.pushButton_7.setText(_translate("Padavan_l1_2", "(в разработке)"))
        self.pushButton_8.setText(_translate("Padavan_l1_2", "..."))
        self.label_2.setText(_translate("Padavan_l1_2", "Генерал-джедай"))
        self.label_6.setText(_translate("Padavan_l1_2", "Находится в разработке"))
        self.label_5.setText(_translate("Padavan_l1_2", "Мастер"))
        self.label_4.setText(_translate("Padavan_l1_2", "Находится в разработке"))


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    Padavan_l1_2 = QtWidgets.QMainWindow()
    ui = Ui_Level_Menu()
    ui.setupUi(Padavan_l1_2)
    Padavan_l1_2.show()
    sys.exit(app.exec())
